artifactory/commands/utils/templateutils.go

```diff
@@ -38,6 +38,32 @@ func ConvertTemplateToMap(tuc TemplateUserCommand) (map[string]interface{}, erro
 	return configMap, errorutils.CheckError(err)
 }
 
+func ConvertTemplateToMaps(templateUserCommand TemplateUserCommand) ([]map[string]interface{}, error) {
+	content, err := fileutils.ReadFile(templateUserCommand.TemplatePath())
+	if errorutils.CheckError(err) != nil {
+		return nil, err
+	}
+	// Replace vars string-by-string if needed
+	if len(templateUserCommand.Vars()) > 0 {
+		templateVars := coreutils.SpecVarsStringToMap(templateUserCommand.Vars())
+		content = coreutils.ReplaceVars(content, templateVars)
+	}
+
+	var multiRepoCreateEntities []map[string]interface{}
+	err = json.Unmarshal(content, &multiRepoCreateEntities)
+	if err == nil {
```

Comment:
can we detect if the error is only about parsing an object instead of array and continue if the error is about parsing single json object else return the error?